README.md

```diff
@@ -1,16 +1,25 @@
-# exeiac
-
-execute infra code smartly by :
-- trying to solve dependency issues
-- being transparent on how to deploy a part of your infra (no matter if you
-  use terraform or ansible or... you will use the same interface)
-- permit proper shortcut that don't break dependency tree even if it's not
-  easily managed by your tool
-
-exeiac use the brick metaphor. It describes infra as a set of bricks. A brick
-is a part of infra (the real infra element, the code that describes it, the code
-that permits to execute it). It can be a terraform state, an ansible playbook,
-a helm chart...
+# Exeiac
+
+## Description
+`exeiac` is a tool that enables infrastructure folks to handle several
+different provisionning IaC (Infrastructure as Code) tools, under a
+single CLI, and helps solving some recurrent paintpoints with IaC.
+
+It follows the brick convention, which describes an infrastructure as
+a set of bricks.  A brick is a piece of infrastructure, it can be the
```

Comment:
. A brick is a piece of infrastructure (ie. actual infrastructure element, the code that describes it, a piece
code that allows its execution)

Remove the double  space at start. Anyway a brick is the 3 at the same time. You can remove the parentheses,  but you can't say it can be the real infra element or the code that describes it.